src/runtime_src/core/pcie/driver/linux/include/xocl_ioctl.h

```diff
@@ -154,6 +156,8 @@ enum drm_xocl_ops {
 	DRM_XOCL_USER_INTR,
 	/* Read xclbin/axlf */
 	DRM_XOCL_READ_AXLF,
+	/* Register xclbin/axlf */
+	DRM_XOCL_REGISTER_AXLF,
```

Comment:
I'd expect to see a new IOCTL for hw context, we don't need it?

Reply:
I have added those ioctl under **enum drm_xocl_ctx_code**
enum drm_xocl_ctx_code {
        XOCL_CTX_OP_ALLOC_CTX = 0,
        XOCL_CTX_OP_FREE_CTX,
        XOCL_CTX_OP_OPEN_UCU_FD,
        **_XOCL_CTX_OP_ALLOC_HW_CTX, 
        XOCL_CTX_OP_FREE_HW_CTX,
        XOCL_CTX_OP_OPEN_CU_CTX,
        XOCL_CTX_OP_FREE_CU_CTX_**
};